example/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  docker-irestarter:
+    image: zetifi/docker-irestarter:latest
+    restart: always
+    volumes:
+      - ./irestarter-test/live:/etc/letsencrypt:ro # Local file or directory to monitor for changes
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      - MONITOR_FILE=/etc/letsencrypt
+      - SIGNAL=SIGHUP # Optional signal to pass to main container process, rather than restarting the container
```

Comment:
The original and still default use is for `docker restart`

We found that there were cases where we wanted to pass a specific signal to the main container process rather than restarting it.

For example, [passing SIGHUP to Vault](https://support.hashicorp.com/hc/en-us/articles/5767318985107-Vault-SIGHUP-Behavior) will trigger the Vault server process to reload certificates without having to go through the Vault seal and unseal process.

Reply:
By using the SIGNAL environment variable, any valid Linux signal could be passed.